index.js

```diff
@@ -24,24 +23,22 @@ function printTree(
     if (printNode.length === 2) {
       printNode(tree, `${branch}${branchHead}`);
     } else {
-      console.log(`${branch}${branchHead}${printNode(tree)}`);
+      console.log(`${branch}${branchHead}${printNode(tree) || ''}`);
```

Comment:
This is so we don't `console.log` `null` or `undefined` values

Reply:
Shouldn't that be up to the consumer?

Reply:
Just had another think about this - and I think that the original behaviour was better. I can't think of a valid reason why a user would want to print out undefined/null for the branch - if they are it's likely because they have a bug - so the behaviour isn't that important - but I think that printing out undefined/null leads to a better debugging experience for the consumer, so better to leave it in!

Reply:
How would that work?

Reply:
That makes sense. I've pushed up a different version.

The original version relied on the arity of the `printNode` function. This version relies on the return value of `printNode`. If a string is returned from `printNode`, log it. If `undefined` is returned from `printNode`, assume it has already been logged.